src/Transition/CreateNewIDFUsingRulesV24_2_0.f90

```diff
@@ -485,6 +571,97 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
 
               ! If your original object starts with Z, insert the rules here
 
+!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+              CASE('ZONEHVAC:TERMINALUNIT:VARIABLEREFRIGERANTFLOW')
+                isVariableVolume = .FALSE.
+                CALL GetNewObjectDefInIDD(ObjectName, NwNumArgs, NwAorN, NwReqFld, NwObjMinFlds, NwFldNames, NwFldDefaults, NwFldUnits)
+                nodiff = .false.
+                OutArgs(1:13) = InArgs(1:13)
+                IF (SameString(InArgs(14), 'FAN:VARIABLEVOLUME')) THEN
+                  isVariableVolume = .TRUE.
+                  OutArgs(14) = 'FAN:SYSTEMMODEL'
+                  ! create a new fan object with a new name
+                  OutArgs(15) = TRIM(InArgs(15)) // '_sysModel'
+                  sysFanName = TRIM(InArgs(15)) // '_sysModel'
```

Comment:
Now that you have deleted the old fan you can revert changing the VRF fan name using `// '_sysModel'` because if there is a report variable with this fan's name it will no longer work.

Reply:
makes sense. I will use the old fan name here.